slink-main/index2.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const config = require('./config');
+const app = require('applicant');
+const processor = require('processor');
+const sr = require('smartrecruiters');
+const introduction = require('introduction');
+// const activation = require('activation');
```

Comment:
I'd remove the commented code

Reply:
It will eventually be un-commented.  I’m concentrating on getting the function to run properly and didn’t want too much noise.